### DISCUSSION TITLE
How to report a bug in Safe Exam Browser

### DISCUSSION THREAD

**Discussion by makbenamor** [SafeExamBrowser/seb-win-refactoring#642] [Q&A]
I want to report a severe bug in Safe Exam Browser in a private email or chat, how can I do this

**Comment by dbuechel** [comment]
In case the issue you want to report is a bug, please feel free to create a new bug report: https://github.com/SafeExamBrowser/seb-win-refactoring/issues/new/choose. In case you'd like to disclose a security vulnerability or something related, you can reach us via e-mail under "info" at "safeexambrowser" dot "org".
